index.js

```diff
@@ -157,9 +157,8 @@ Client.prototype.validateXmlrpcVersion = function( callback ) {
 					"XML-RPC extensions for Gilded WordPress are not installed." ) );
 			}
 			if ( !error.code ) {
-				return callback( new Error( "Unknown error. " +
-					"Please ensure that your database server is running " +
-					"and WordPress is functioning properly." ) );
+				error.message += "Please ensure that your database server is running " +
```

Comment:
This probably needs some whitespace between the existing message and the custom message. Maybe just start this string with a newline?